app/assets/javascripts/angular/controllers/users/landing_page.js.erb

```diff
@@ -0,0 +1,4 @@
+app.controller('LandingPageCtrl', ['$scope', '$http', '$location', function($scope, $http, $location){
```

Comment:
if it's empty, perhaps better to add dependencies later -- when/if you need them

```js
app.controller('LandingPageCtrl', [function(){
}]);
```